### ISSUE TITLE
No link from AlleyCAT to Plover, couldn't find the plugin.

### ISSUE THREAD

**Issue by flamenco108** [sammdot/alleycat#1]
I don't know if it should be posted here:
Just after I read (17th Aug 2022) the news on Mirabai's Opensteno blog, immediately I had downloaded the newest/latest Plover and AlleyCAT to my Xubuntu `Linux szmirlap 5.15.0-46-generic #49~20.04.1-Ubuntu SMP Thu Aug 4 19:15:44 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux`.
I started Plover, then AlleyCAT. Could not find in Plugin's Manager the plugin for AlleyCAT (nothing like `alleycat-link`).
Plover works great.
AlleyCAT works great.
But they don't work together.
I'm not a programmer, just a user.

**Comment by sammdot** [comment]
I've already added the AlleyCAT Link plugin to the registry. It requires Plover 4.0.0-dev10+ and  Python 3.7+, if that makes a difference. And it's just called `alleycat-link`, so it should show up at the very top of the list of plugins.

**Comment by flamenco108** [comment]
Below the printscreen from my Linux, list of plugins etc. if it might help.
![2022-08-20-plover-alleycat](https://user-images.githubusercontent.com/5072952/185761321-f62d2d48-8b33-4094-8c48-cd4176bb2963.jpg)
Thanks in advance for help! :)

**Comment by 7632416** [comment]
Yes, EXCITING, so THANK YOU, SAMMDOT.  Your project has such potential, but where's the plug-in to provide the link between Plover and AlleyCAT?

**Comment by flamenco108** [comment]
I would really like to know, how [@Codebreakerblue](https://github.com/sammdot/alleycat/issues?q=is%3Aissue+is%3Aopen+author%3ACodebreakerblue) had installed it properly, so he/she has a link to Plover.
I did also `pip install alleycat-link` on my Linux and on Windows, but result is still the same.
Below parameters on Linux:
```
flamenco@szmirlap:~$ bin/Plover/plover-4.0.0.dev12-x86_64.AppImage --version
Plover 4.0.0.dev12
flamenco@szmirlap:~$ python3 --version
Python 3.8.10
flamenco@szmirlap:~$ python --version
Python 2.7.18
flamenco@szmirlap:~$
```
On Windows:
```
C:\Users\flamenco>python --version
Python 3.10.6
C:\Users\flamenco>plover --version
Plover 4.0.0.dev12
```

**Comment by 7632416** [comment]
Greetings, SAMMDOT. Thanks again for a long-awaited and potentially very useful plug-in app. Do you have any news for us about the link? I've tried everything I can, with no luck.

**Comment by this-duck** [comment]
Salutations, SAMMDOT.  I'm having the same issue as above. 
Plover 4.0.0.dev12. 
Python 3.11.0. 
Keenly interested in giving this a go. (Incidentally, I couldn't get the web version to work either. I could write in the text field, but nothing comes up in the steno notes.)
